themes/ctl-histologylab/static/js/src/tiles.js

```diff
@@ -37,6 +32,9 @@ $(document).ready(function() {
     new L.Control.MiniMap(smallLayer, {
         toggleDisplay: true,
         height: 200,
-        width: 200}).addTo(map)
-
+        width: 200,
+        mapOptions: {
+            panControl: false,
+            maxBounds: [[-90, -200], [90, 200]]
```

Comment:
I'm not sure if this is the cause...but there's a "bounce" now as the slide hits the maxBounds. I'm fine merging this, but we might want to adjust the behavior.